net/FlatBuffers/Table.cs

```diff
@@ -190,11 +190,10 @@ public static int CompareStrings(int offset_1, int offset_2, ByteBuffer bb)
         }
 
         // Compare string from the ByteBuffer with the string object
-        public static int CompareStrings(int offset_1, byte[] key, ByteBuffer bb)
+        public static int CompareStrings(int offset_1, byte[] key, int len_2, ByteBuffer bb)
```

Comment:
len_2 is not that descriptive, can you say key_length?

Reply:
Agreed `key_length` is better.